server/paths/refer.ts

```diff
@@ -13,6 +13,8 @@ const personPath = peoplePathBase.path(':prisonNumber')
 const referralsPath = path('/referrals')
 const showReferralPath = referralsPath.path(':referralId')
 const referralPersonPath = showReferralPath.path('person')
+const programmeHistoryBasePath = showReferralPath.path('programme-history')
+const newProgrammeHistoryPath = programmeHistoryBasePath.path('new')
```

Comment:
I'm a little less sure about the course participations vs programme history question here - I think `/programme-history` is clearer for the user, but `/programme-history/new` feels a little strange when the entity we're creating is a course participation. I've gone for `/course-participations` in the branch I'm currently working on, but I'm curious what you and @Gweaton think on this one!

Reply:
Let's go with `/programme-history/new` (and `/programme-history`) - just run this by George in an unrelated chat!